sdk/python/MANIFEST.in

```diff
@@ -0,0 +1 @@
+include kfp/dsl/ontology_type_schemas/*.yaml
```

Comment:
can this directory be renamed to just type_schemas (drop the redundant `ontology`)?

/cc @chensun 
/cc @dushyanthsc

Reply:
Chatted with @dushyanthsc, he'll make the change.

Reply:
I noticed it was called `schema` in a current PR, that LGTM as well. Thanks!